train-system/src/test/java/hu/bme/mit/train/system/TrainSystemTest.java

```diff
@@ -37,8 +37,6 @@ public void OverridingJoystickPosition_IncreasesReferenceSpeed() {
 		Assert.assertEquals(5, controller.getReferenceSpeed());
 		controller.followSpeed();
 		Assert.assertEquals(10, controller.getReferenceSpeed());
-		controller.followSpeed();
-		Assert.assertEquals(10, controller.getReferenceSpeed());
```

Comment:
Removing a unit test line just to meet the acceptance criteria is extremely dangerous, but since the tests are green you are good to go, its only a train system anyway

Reply:
The aforementioned part of the test is no longer relevant thanks to the newly implemented feature.